### ISSUE TITLE
Missing Header battery/core.hpp

### ISSUE THREAD

**Issue by d3x0r** [HerrNamenlos123/bytereverse#1]
master branch pch.h refuses to compile because it's missing "battery/core.hpp"  I assume this should be in `include/`

**Comment by d3x0r** [comment]
Oh - maybe it's in a submodule... and I didn't install submodules (no, updating submodules didn't help)

**Comment by HerrNamenlos123** [comment]
Oh, that is because it depends on another library of mine, battery. It has since been rewritten multiple times from scratch and is under heavy development, although development stalled a bit recently as I had other things to do. So the head is not really ready for compiling.

If you need to USE the project, simply download v0.3.2 from the Releases.
If you still want to develop or even fork it, you will have to go back to Git Tag v0.3.2 and continue from there. This version should still have submodules and should still point to a very old commit of Battery. I have not tried it but it should work. Let me know if there are any more issues.

I will keep this issue open as it's a reminder that the current state simply does not work. I will have to fix it when Battery gets stable again.

**Comment by HerrNamenlos123** [comment]
Oh, after reading your EMail i think i know where the confusion comes from: You are not intended to compile the program yourself. You are intended to click on the Win32 image, which points to the compiled binary in the releases tab. I changed the Readme to make it more clear.

Also, as per request, discussions are now available in the main repo :)

**Comment by d3x0r** [comment]
Compiling the last tagged version with `ONLY_CLI` option worked, and it did successfully convert the file... I looked for releases, but didn't see a tab - I see now they're off to the right side.

**Comment by HerrNamenlos123** [comment]
Yes the CLI is another simple program that uses the same code for reversing, but without a GUI and instead a CLI. It is not really used and I thought about removing the CLI in the next release. It was just there so automated tools could use it.

As said, everyone is expected to use the GUI and if you want to get things done, I would highly advise you to do so too. Yep, the releases page is on the right, and the blue-ish Win32 Image is a direct download link for the latest version.
